source/developer/webhooks-incoming.rst

```diff
@@ -40,7 +40,7 @@ If you're running cURL on Windows, you need to wrap the payload with double quot
 
```

Comment:
This line say that you have to wrap the payload with `"` instead of `'`, and the example doesn't do that.

Reply:
@crspeller ^

Reply:
Removed the bad text.